src/lib/journey/callbacks/confirmation/confirmation.svelte

```diff
@@ -112,38 +111,41 @@
   }
 </script>
 
-{#if !stepMetadata?.derived.isStepSelfSubmittable}
-  {#if options.length > 1}
-    <Select
-      isFirstInvalidInput={callbackMetadata?.derived.isFirstInvalidInput || false}
-      isRequired={false}
-      key={inputName}
-      {label}
-      onChange={setOptionValue}
-      {options}
-    />
+<!-- Only render confirmation if NOT currently on 'OneTimePassword' stage -->
```

Comment:
Most of this remains the same with the slight change of the checking for "OneTimePassword" as the stage name. If this check is true, then we don't render the ConfirmationCallback's UI.